src/Uno.UI/UI/Xaml/Controls/ListViewBase/ListViewBaseSource.iOS.cs

```diff
@@ -252,6 +252,13 @@ public override UICollectionViewCell GetCell(UICollectionView collectionView, NS
 					// Normally this happens when the SelectorItem.Content is set, but there's an edge case where after a refresh, a
 					// container can be dequeued which happens to have had exactly the same DataContext as the new item.
 					cell.ClearMeasuredSize();
+
+					// Ensure ClippedFrame from a previous recycled item doesn't persist which can happen in some cases,
+					// and cause it to be clipped when either axis was smaller.
+					if (cell.Content is { } contentControl)
+					{
+						contentControl.ClippedFrame = null;
+					}
```

Comment:
@Xiaoy312  Very nice! It is something we can catch in a test? Or if we can't, can we assert that the clipping is removed from recycled items maybe?